eventdisplay/tracks/FairEveTracks.h

```diff
@@ -64,13 +65,18 @@ class FairEveTracks
     TObjArray *GetTracksList() const { return fEveTrList; };
     /**
      *
-     * @param tr
+     * @param groupName - name of the track group
+     * @param color - group color
      * @return track group based on parameter
      */
-    virtual TEveTrackList *GetTrackGroup(void *tr);
+    TEveTrackList *GetTrackGroup(TString groupName, Color_t color);
```

Comment:
```suggestion
    TEveTrackList *GetTrackGroup(TString groupName, Color_t color) const;
```

*constness*: Depends on if we can get rid of the `fTrPropagator` and `fTrackGroup` member variables, see my comments in the cxx file.

```suggestion
    std::unique_ptr<TEveTrackList> GetTrackGroup(TString groupName, Color_t color);
```
Ownership:
* If we remove `fTrackGroup`, you are returning an owning pointer, see 
* If we do not remove `fTrackGroup` (and you intend `FairEveTracks` to own it), you need to
   * either make it a `std::unique_ptr<TEveTrackList>`, or
   * call delete on it in the dtor, otherwise it leaks.
* The same story with `fTrPropagator`, it is leaking.

Reply:
> If we remove fTrackGroup, you are returning an owning pointer

I actually missed the `GetTracksList()->Add(TrackGroup);` so nevermind my comment about the return value, raw pointer should be fine.

Reply:
One more remark - i was trying to add TEveTrackPropagator to list to properly remove them.
However if I try to remove them by hand I get "free()" error, it seems that they are somehow removed by gEve.
I write my own propagator to check if they are removed - and they are somehow removed by gEve.